src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -58,6 +58,20 @@ public DebugService(PowerShellContext powerShellContext)
         #region Public Methods
 
         /// <summary>
+        /// Disables all breakpoints in the runspace.
+        /// </summary>
+        /// <returns></returns>
+        public async Task DisableAllBreakpoints()
```

Comment:
Man, I hate having to do that.  There needs to be some way to refer to a setting string for the extension in a debug config parameter...